### PR TITLE
fix: #6168, update error messages, yarn.lock

### PR DESCRIPTION
**Pull request by jhockett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/src/attach-backend.js [hunk 1]
```diff
@@ -27,25 +27,18 @@ async function run(context) {
...
-          try {
-            await adminLoginFlow(context, appId, envName, res.region);
-          } catch (e) {
-            context.print.error(`Failed to authenticate: ${e.message || 'Unknown error occurred.'}`);
-          }
-        }
+  if (!useProfile && (!configLevel || configLevel === 'amplifyAdmin') && appId) {
+    const res = await isAmplifyAdminApp(appId);
+    isAdminApp = res.isAdminApp;
+    if (isAdminApp) {
```
Comment (kaustavghosh06) [inline packages/amplify-provider-awscloudformation/src/attach-backend.js:33]: What's the backup option here - if a customer is using cloud9/EC2 instances and have admin app enabled? Can they still use the AWS profile instead?

Comment (jhockett) [reply packages/amplify-provider-awscloudformation/src/attach-backend.js:33]: It's not ideal, but you can workaround by changing the `configLevel` to `"project"` in the `<root>/amplify/.config/local-aws-info.json` file.

Comment (jhockett) [reply packages/amplify-provider-awscloudformation/src/attach-backend.js:33]: A larger PR to address fallback scenarios is coming, but I didn't want to hold this fix up on it.

Comment (kaustavghosh06) [reply packages/amplify-provider-awscloudformation/src/attach-backend.js:33]: Ah! Got it.

**Review by kaustavghosh06 — APPROVED** [review]
LGTM.

**Pull request merged**